src/Plugin/Block/AlertBannerBlock.php

```diff
@@ -28,20 +30,6 @@ class AlertBannerBlock extends BlockBase implements ContainerFactoryPluginInterf
    */
   protected $entityTypeManager;
 
-  /**
```

Comment:
Removing these already, when only deprecated, is also to highlight the change? (and I assume they aren't used in these implementations any more).

Reply:
There not actually set anymore by the alert banner, though I could re-add them if needed.

Reply:
:+1:

Reply:
So I need to add this trait https://www.drupal.org/about/core/policies/core-change-policies/how-to-deprecate#how-injected-service